fix: file apis (#226)
smartmind-team/thanosql-docs: docs/en/how-to_guides/ThanoSQL_connecting/rest/rest_api_thanosql_file.md

# **File APIs**

You can use File APIs to remotely send and upload files to your Workspace storage and insert them into any of your table within the database. The default root folder for all user files uploaded by this APIs are set to 'drive/'.

## __`GET` /file/__

> Returns a list of files and folders that match the given path. The path shoulde include root foolder 'drive/' and it can be expressed using a regular expression.

Reviewer: ```suggestion
Returns a list of files and folders that match the given path. The path should include root folder 'drive/' and it can be expressed using a regular expression.
```

## __`POST` /file/__

To upload a file to the drive/ directory in the Workspace, use the following methods to send the file.

> When the 'dir={dir}' is added to the URL, the file will be uploaded to the drive/{dir}/.

Reviewer: ```suggestion
When the query param 'dir={dir}' is added to the URL, the file will be uploaded to the drive/{dir}/.
```

## __`DELETE` /file/__

> To delete a file from the drive/ directory in the Workspace, use the following methods.

Reviewer: ```suggestion
To delete a file from the drive/ directory in the Workspace, use the following method.
```
